test/testdrive/joins.td

```diff
@@ -225,13 +221,3 @@ num name num mod
 <null> <null> 0 even
 <null> <null> 1 odd
 <null> <null> 2 even
-
-> CREATE VIEW test15 AS
```

Comment:
I am still in favor of having at least one test in testdrive that is a join between a materialized and an unmaterialized view/source.

Reply:
Oh, I didn't realize that's what this was intended to be testing. You mean the fact that joining one materialized and one unmaterialized source results in an error?

Reply:
Ok, restored this block and merging!